engine/src/test/scala/cromwell/RetryableCallsSpec.scala

```diff
@@ -27,25 +25,27 @@ class RetryableCallsSpec extends CromwellTestkitSpec with Mockito {
     }
   }
 
+  val customizedLocalBackendOptions =
+    """
+      |{
+      |  "backend": "customizedLocalBackend"
+      |}"""".stripMargin
```

Comment:
I'm not crazy... there are definitely too many quotes, right?  My brain parser is reading this as

```
{
  "backend": "customizedLocalBackend"
}"
```